experiments/integrated/fluxnet/run_fluxnet.jl

```diff
@@ -136,6 +136,9 @@ canopy_component_types = (;
 autotrophic_respiration_args =
     (; parameters = AutotrophicRespirationParameters(FT))
 # Set up radiative transfer
+if @isdefined(χl)
+    ld = ld_from_χl(χl)
+end
```

Comment:
This is a rather fragile way to define `ld`. If you want to control this behavior, best to be specific about it by adding a flag that determines if ld should computed from chi or not.